stable/storageos-operator/README.md

```diff
@@ -148,9 +148,9 @@ Operator chart and their default values.
 
 Parameter | Description | Default
 --------- | ----------- | -------
-`operator.image.repository` | StorageOSCluster container image repository | `storageos/cluster-operator`
-`operator.image.tag` | StorageOSCluster container image tag | `1.1.0`
-`operator.image.pullPolicy` | StorageOSCluster container image pull policy | `IfNotPresent`
+`operator.image.repository` | StorageOS Operator container image repository | `storageos/cluster-operator`
+`operator.image.tag` | StorageOS Operator container image tag | `1.2.0`
+`operator.image.pullPolicy` | StorageOS Operator container image pull policy | `IfNotPresent`
 `podSecurityPolicy.enabled` | If true, create & use PodSecurityPolicy resources | `false`
 `podSecurityPolicy.annotations` | Specify pod annotations in the pod security policy | `{}`
 `cluster.create` | If true, auto-create the StorageOS cluster | `true`
```

Comment:
Can you add a doc line for `cluster.csi.deploymentStrategy`?  I did it at the end of the block:  
https://github.com/rancher/charts/pull/163/files#diff-e72916d3373dc9723fce013bd3357dbcR176